# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../e2fsprogs/powerpc64.patch
# Copyright (C) 2004 - 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./lib/blkid/blkid_types.h.in.vanilla	2005-10-27 01:03:35.146713072 +0300
+++ ./lib/blkid/blkid_types.h.in	2005-10-27 01:04:29.142625760 +0300
@@ -3,7 +3,7 @@
  * everything we need.  (cross fingers)  Other header files may have 
  * also defined the types that we need.
  */
-#if (!defined(_LINUX_TYPES_H) && !defined(_BLKID_TYPES_H) && \
+#if (!defined(_LINUX_TYPES_H) && !defined(_BLKID_TYPES_H) && !defined(_PPC64_TYPES_H) && \
 	!defined(_EXT2_TYPES_H))
 #define _BLKID_TYPES_H

--- ./lib/ext2fs/ext2_types.h.in.vanilla	2005-10-27 00:59:45.932682280 +0300
+++ ./lib/ext2fs/ext2_types.h.in	2005-10-27 01:02:44.810644824 +0300
@@ -3,7 +3,7 @@
  * everything we need.  (cross fingers)  Other header files may have 
  * also defined the types that we need.
  */
-#if (!defined(_LINUX_TYPES_H) && !defined(_BLKID_TYPES_H) && \
+#if (!defined(_LINUX_TYPES_H) && !defined(_BLKID_TYPES_H) && !defined(_PPC64_TYPES_H) && \
 	!defined(_EXT2_TYPES_H))
 #define _EXT2_TYPES_H
 
